tests/containers/podman_ipv6.pm

```diff
@@ -0,0 +1,89 @@
+# SUSE's openQA tests
+#
+# Copyright 2023-2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman, netavark, aardvark
+# Summary: Test podman IPv6
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use version_utils qw(package_version_cmp is_transactional is_jeos is_leap is_sle_micro is_leap_micro is_sle is_microos is_public_cloud);
+use containers::common qw(install_packages);
+use publiccloud::utils 'is_gce';
+
+# clean up routine only for systems that run CNI as default network backend
+sub _cleanup {
+    my $podman = shift->containers_factory('podman');
+    select_console 'log-console';
+    $podman->cleanup_system_host();
+}
+
+sub run {
+    my ($self, $args) = @_;
+
+    die('Please set PUBLIC_CLOUD_GCE_STACK_TYPE=IPV4_IPV6') if (is_gce && !check_var('PUBLIC_CLOUD_GCE_STACK_TYPE', 'IPV4_IPV6'));
+
+    my $image = 'registry.opensuse.org/opensuse/leap:latest';
+
+    select_serial_terminal;
+    my $podman = $self->containers_factory('podman');
+    install_packages('bind-utils');
+
+    my $host_if = script_output('ip -6 route show default | awk "{print \$5; exit}"');
+    record_info('HOST IF', $host_if);
+    my $host_addr = script_output("ip -6 addr show $host_if scope global | grep -oP 'inet6 \\K[^/]+' | head -n1");
+    record_info('HOST ADDR', $host_addr);
+    my $host_gw = script_output('ip -6 route show default | awk "{print \$3}"');
+    record_info('HOST GW', $host_gw);
+    my $subnet = $host_addr;
+    $subnet =~ s/::/:1:0\/112/g;
+    record_info('SUBNET', $subnet);
+    my $cont_addr = $subnet;
+    $cont_addr =~ s/:0\/112/:2/g;
+    record_info('CONT ADDR', $cont_addr);
+
+    # Test host IPv6 connectivity
+    validate_script_output('curl -sSf6 https://wtfismyip.com/text', sub { m/$host_addr/g });
```

Comment:
I don't have better suggestion actually , but just for the record - I am a bit worry about the fact that we relying on external resource here which is not under our control :thinking:

Reply:
We could replace it with a simple `ping -c 2 opensuse.org`

Reply:
isn't all what we doing after that line proves that there is connectivity ? maybe we can just skip such check ?

Reply:
I would like to check the IPv6 connectivity, so I will simplify it.

Reply:
LGTM.